Default (Linux).sublime-keymap

```diff
@@ -3,6 +3,7 @@
     { "keys": ["ctrl+alt+r"], "command": "replace_modified_part" },
     { "keys": ["ctrl+alt+d"], "command": "show_diff" },
     { "keys": ["ctrl+alt+u"], "command": "uncommitted_files" },
+    { "keys": ["ctrl+alt+t"], "command": "toggle_highlight_changes" },
```

Comment:
I don't want to use `ctrl+alt+t` for this, because on Linux it's a standard shortcut for Terminal app.
Do you think `ctrl+alt+h` will be OK for Win and Mac?